upload/views/test_results.py

```diff
@@ -39,13 +39,13 @@ def has_permission(self, request, view):
 class UploadSerializer(serializers.Serializer):
     commit = serializers.CharField(required=True)
     slug = serializers.CharField(required=True)
+    service = serializers.CharField(required=False)  # git_service
```

Comment:
How come TA can't use `git_service`?

Reply:
Since it's already live in prod I figured it would be a whole thing to switch over to a different var name - what do you think @joseph-sentry ?